pagerduty/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fbbb4a11-4a8f-4911-bdf7-bd867d9bdfb2",
+  "app_id": "pagerduty",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "README.md#Description",
```

Comment:
This should be an inline short description of your app (it can't reference a section of the README)